schema/shapes/base-stroke.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Base Stroke",
+    "description": "Common properties for stroke styles",
+    "allOf": [
+        {
+            "type": "object",
+            "properties": {
+                "lc": {
+                    "title": "Line Cap",
+                    "$ref": "#/$defs/constants/line-cap",
+                    "default": 2
+                },
+                "lj": {
+                    "title": "Line Join",
+                    "$ref": "#/$defs/constants/line-join",
+                    "default": 2
+                },
+                "ml": {
+                    "title": "Miter Limit",
+                    "type": "number",
+                    "default": 0
+                },
+                "ml2": {
+                    "title": "Miter Limit",
+                    "description": "Animatable alternative to ml",
+                    "$ref": "#/$defs/properties/scalar-property"
+                },
```

Comment:
Oh, I did not know this was a thing, seems kinda confusing to have separate props for animatable vs. static.

Not the right time to change, but maybe we can revisit in the future where the "ml" property could be either static or animatable.  It should be fairly easy to discriminate at parse time, no?

Come to think of it, that approach could make sense for all non-keyframed animatable props, e.g.

```
"foo": $fooval
```
vs
```
"foo": { "a": 0, "k": $fooval}
```

The former is significantly more compact, and there are lots of non-keyframed props in all Lottie animations that would benefit.